pkg/apis/workspaces/v1alpha2/projects.go

```diff
@@ -109,7 +109,8 @@ type GitLikeProjectSource struct {
 	// +optional
 	CheckoutFrom *CheckoutFrom `json:"checkoutFrom,omitempty"`
 
-	// The remotes map which should be initialized in the git project. Must have at least one remote configured
+	// The remotes map which should be initialized in the git project.
+	// Projects must have at least one remote configured while StarterProjects can only have at most one remote configured.
```

Comment:
Should this comment be added to the `StarterProject` and `Project` structs as well (over ProjectSource maybe)?

Reply:
WDYM?  should it already included? since `StarterProject/Projects.ProjectSource` references `GitLikeProjectSource`